.github/workflows/integration.yml

```diff
@@ -42,3 +43,16 @@ jobs:
             yarn build:contracts
             yarn build:contracts:ovm
             yarn test:integration
+
+      # Examples Tests
+      - name: Test & deploy hardhat-example on hardhat (regression)
+        working-directory: ./examples/hardhat
+        run: |
+          yarn deploy
```

Comment:
I noticed you are deploying before testing. Is this intentional?

Reply:
you also never compile the contracts with `yarn compile`

Reply:
Compiling happens automatically by hardhat, so no need for us to do it explicitly here